packages/test-interface-criteria/built-types/types/TestDataShape.d.ts

```diff
@@ -20,12 +19,16 @@ export interface NumericNodeConstraint {
   maxinclusive?: number;
 }
 
+export interface BooleanNodeConstraint {
+  '@type': 'test:BooleanNodeConstraint';
+  value: boolean; 
```

Comment:
`value` picked as property as that seems to be the language used in Shex. Comments welcome!